eng/Versions.props

```diff
@@ -35,6 +35,8 @@
       release, increase this number by one.
     -->
     <NETStandardPatchVersion>0</NETStandardPatchVersion>
+    <!-- Restore the .NETFramework reference assemblies from NuGet packages -->
+    <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
```

Comment:
We have patches in other repos to disable this prop in source-build, but I'm not familiar with it.
Example: https://github.com/dotnet/source-build/pull/1218.
@NikolaMilosavljevic @dseefeld @crummel @adaggarwal do you know why?

Reply:
We could further condition it.  FWIW I had to install other things after this initial failure, so I'm not certain about the minimality of this.  Ideally dotnet/runtime would be buildable without pre-requisites but we are further from that now (CMake, Python, VS for C++ on Windows).

Reply:
I do see that most of these packages have a version in  https://github.com/dotnet/source-build-reference-packages/tree/master/src/targetPacks/ILsrc, so I'd be surprised if this is actually a problem for source-build.  Maybe a point-in-time issue that no longer exists?

Reply:
Yeah, if we have a targeting pack for the framework you're using there then we should no longer need the change.  Prior to having those reference packages we would disable full-framework builds and turn this off to prevent pulling them in as prebuilts.